web/stories/Molecules/Button/Feedback.stories.tsx

```diff
@@ -18,25 +18,10 @@ const Template: ComponentStory<typeof PrimaryButton> = ({ children, ...args }) =
   <PrimaryButton {...args}>{children}</PrimaryButton>
 );
 
-export const Regular = Template.bind({});
-
-Regular.args = {
-  children: (
-    <>
-      <ButtonIcon svgFilename="chat-processing" />
-      <span className="text-left">Text Here</span>
-    </>
-  ),
-  isColor: "accent-cool-lightest",
-  isFullWidthOnDesktop: true,
-  isTextAlignedLeft: true,
```

Comment:
[dust] I believe this property was exclusively used for regular feedback design. As we are removing this styling, it makes sense to delete this property from the generic button component as well.

Reply:
I believe the same principle should also apply to the 'isFullWidthOnDesktop' property.

Reply:
So it appears isFullWidthOnDesktop is used in a number of places still 
<img width="1356" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/858902d8-7d92-4df9-8015-2debb8f618c6">

Reply:
I removed isTextLeftAligned